src/pages/[...slug].tsx

```diff
@@ -162,6 +189,37 @@ export const query = graphql`
   }
 `
 
-Page.displayName = 'Page'
+export const getServerData = async ({
+  params: { slug },
+}: {
+  params: Record<string, string>
+}) => {
+  try {
+    const { execute } = await import('src/server')
+    const { data } = await execute({
+      operationName: querySSR,
+      variables: { slug },
+    })
+
+    return {
+      status: 200,
+      props: data ?? {},
+      headers: {
+        'cache-control': 'public, max-age=0, stale-while-revalidate=31536000',
```

Comment:
what does the `31536000` number mean? What if we extract to a named constant or add a comment?

Reply:
one year in seconds